src/update-changelog.ts

```diff
@@ -37,18 +37,18 @@ async function getCommits(commitHashes: string[]) {
       commitHash,
     ]);
 
-    let matchResults = subject.match(/\(#\d+\)/u);
+    let matchResults = subject.match(/\(#(\d+)\)/u);
     let prNumber: string | undefined;
     let description = subject;
 
-    // Squash & Merge: the commit subject is parsed as `<description> (#<PR ID>)`
     if (matchResults) {
+      // Squash & Merge: the commit subject is parsed as `<description> (#<PR ID>)`
       prNumber = matchResults[1];
```

Comment:
Without the capturing group around `\d+` on L40, `prNumber` would always be `undefined`.